common/lib/xmodule/xmodule/contentstore/mongo.py

```diff
@@ -131,15 +130,20 @@ def find(self, location, throw_on_not_found=True, as_stream=False):
     def export(self, location, output_directory):
         content = self.find(location)
 
+        filename = content.name
         if content.import_path is not None:
             output_directory = output_directory + '/' + os.path.dirname(content.import_path)
 
         if not os.path.exists(output_directory):
             os.makedirs(output_directory)
 
+        # Strip '/' from filename if there are any.
+        if '/' in filename:
```

Comment:
@awaisdar001 Can you try it with other special characters too ? Just to make sure that the things are working as expected with them.